br/pkg/lightning/lightning.go

```diff
@@ -960,8 +961,8 @@ func checkSystemRequirement(cfg *config.Config, dbsMeta []*mydump.MDDatabaseMeta
 				tableTotalSizes = append(tableTotalSizes, tb.TotalSize)
 			}
 		}
-		slices.SortFunc(tableTotalSizes, func(i, j int64) bool {
-			return i > j
+		slices.SortFunc(tableTotalSizes, func(i, j int64) int {
+			return -cmp.Compare(i, j)
```

Comment:
```suggestion
			return cmp.Compare(j, i)
```

Reply:
I think what he wants here is to sort in descending order.

Reply:
the arguments are also swapped. It's about readability.